src/AssetConfig.php

```diff
@@ -180,7 +180,7 @@ public function load($path, $prefix = '')
             } elseif (strpos($section, self::FILTER_PREFIX) === 0) {
                 // filter section.
                 $name = str_replace(self::FILTER_PREFIX, '', $section);
-                $this->filterConfig($name, $values);
+                $this->filterConfig($name, array_merge($this->filterConfig($name), $values));
```

Comment:
I found it might not make sense in some cases. E.g. when I'm using `PostCSS` filter, there are `options`, `path`, `node_path`, `postcss` settings. `options` should be defined in `asset_compress.ini` and the rest settings should be defined in `asset_compress.local.ini`. I think this makes more sense rather than defining all of them in either `asset_compress.ini` or local ini version.